var_mu: 8.0 #2.0
var_muprime: 20.0 #2.0
var_q: 1.0 #1.0
var_qdot: 1.0 #80.0

var_q_d: 2.0 # 1.0
var_qdot_d: 10000000000.0 # 20.0

k_a: 7.0 # 5.4
k_mu : 30.0  #100.0